src/pages/Flags/Debugging.tsx

```diff
@@ -50,14 +70,25 @@ export const Debugging = () => {
               </Menu>
             </Row.Value>
           </Row>
+          <Row alignItems='center'>
+            <Row.Label>Pendo visitor ID</Row.Label>
+            <Row.Value display='flex' gap={4} alignItems='center'>
+              <RawText>{visitorId}</RawText>
+              <IconButton
+                aria-label='Copy'
+                size='sm'
+                icon={<CopyIcon />}
+                onClick={handleCopyClick}
+              />
+            </Row.Value>
+          </Row>
         </Card.Body>
+        <Card.Footer>
+          <Button onClick={() => window.location.reload()} colorScheme='blue'>
```

Comment:
One less function in the stack trace = happier debugging
```suggestion
          <Button onClick={window.location.reload} colorScheme='blue'>
```